articles/azure-monitor/app/tutorial-custom-metrics.md

```diff
@@ -0,0 +1,370 @@
+
+    ```kql
+    customMetrics 
+    | where name == "ReviewPerformed"
+    ```
+
+3. Observe the results display the rating value present in the Review.
+
+## GetMetric
+
+As referenced before, `GetMetric(..)` is the preferred method for sending metrics. In order to make use of this method, we will be performing some changes to the existing code.
+
+When running the sample code, you'll see that no telemetry is being sent from the application right away. A single telemetry item will be sent by around the 60-second mark .
+
+> [!NOTE]
+> GetMetric does not support tracking the last value (i.e. "gauge") or tracking histograms/distributions.
+
+### GetMetric example
+
+1. From the Visual Studio Solution Explorer, locate and open the **HomeController.cs** file.
+
+2. Locate the `CreateReview` method and the code added in the previous [TrackMetric example](./custom-metrics.md#trackmetric-example).
+
+3. Replace the previously added code in _Step 3_ with the following one.
+
+   ```csharp
+   var metric = _telemetryClient.GetMetric("ReviewPerformed");
+   metric.TrackValue(model.Rating);
+   ```
+
+4. Right-click the **AzureCafe** project in Solution Explorer and select **Publish** from the context menu.
+
+    ![The Visual Studio Solution Explorer displays with the Azure Cafe project selected and the Publish context menu item highlighted.](./media/asp-net-core/web_project_publish_context_menu.png "Publish Web App")
+
+5. Select **Publish** to promote the new code to the Azure App Service.
+
+    ![The AzureCafe publish profile displays with the Publish button highlighted.](./media/asp-net-core/publish_profile.png "Publish profile")
+
+6. Once the publish has succeeded, a new browser window opens to the Azure Cafe web application.
+
+    ![The Azure Cafe web application displays.](./media/asp-net-core/azure_cafe_index.png "Azure Cafe web application")
+
+7. Perform various activities in the web application to generate some telemetry.
+
+   1. Select **Details** next to a Cafe to view its menu and reviews.
+
+        ![A portion of the Azure Cafe list displays with the Details button highlighted.](./media/asp-net-core/cafe_details_button.png "Azure Cafe Details")
+
+   2. On the Cafe screen, select the **Reviews** tab to view and add reviews. Select the **Add review** button to add a review.
+
+        ![The Cafe details screen displays with the Add review button highlighted.](./media/asp-net-core/cafe_add_review_button.png "Add review")
+
+   3. On the Create a review dialog, enter a name, rating, comments, and upload a photo for the review. Once completed, select **Add review**.
+
+        ![The Create a review dialog displays.](./media/asp-net-core/create_a_review_dialog.png "Create a review")
+
+   4. Repeat adding reviews as desired to generate additional telemetry.
+
+### View logs in Application Insights
```

Comment:
For viewing metrics, we should show both the Logs one, and using the metric explorer UI based on as well.